PATTERNS.md

```diff
@@ -197,3 +198,52 @@ myQueue.add({msg: 'Specific Error'}, {
  }
 });
 ```
+
+Async multiple-repo job processor
```

Comment:
I am a bit unsure about the title, maybe it is not so clear. So the pattern is actually about "manually fetching jobs" from the queue, maybe that would be better, then the user may use it for many different scenarios.

Reply:
totally